src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -46,7 +47,12 @@ def _post(self, request: Request):
         version = request.GET.get("version") or "1"
         set_tag("relay_protocol_version", version)
 
-        if version in ["2", "3"]:
+        if version == "3":
```

Comment:
If we want to be very correct, this branch should only be entered for V3 requests without `no_cache`. See the implementation in Relay:
https://github.com/getsentry/relay/blob/7e63150f486fa6cd35db599aef6287040601f8e7/relay-server/src/endpoints/project_configs.rs#L132